src/main/java/uk/gov/digital/ho/hocs/queue/common/QueueListener.java

```diff
@@ -35,4 +35,20 @@ public void onComplaintEvent(String message, @Header("MessageId") String message
         }
     }
 
+    @SqsListener(value = "${aws.sqs.case-migrator.url}", deletionPolicy = SqsMessageDeletionPolicy.ON_SUCCESS)
```

Comment:
Won't be required as using the main queue.

Reply:
Fixed